internal/core/graphics/brush.rs

```diff
@@ -61,6 +61,36 @@ impl Brush {
             Brush::LinearGradient(_) => false,
         }
     }
+
+    /// Returns a new version of this brush that has the brightness increased
+    /// by the specified factor. This is done by calling [`Color::brighter`] on
+    /// all the color of this brush.
+    #[must_use]
+    pub fn brighter(&self, factor: f32) -> Self {
+        match self {
+            Brush::SolidColor(c) => Brush::SolidColor(c.brighter(factor)),
+            Brush::LinearGradient(g) => Brush::LinearGradient(LinearGradientBrush(
+                g.0.iter()
+                    .map(|s| GradientStop { color: s.color.brighter(factor), position: s.position })
+                    .collect(),
+            )),
```

Comment:
I think it's cleaner to not rely on the implementation of the gradient inside brush and use the API instead:

```suggestion
            Brush::LinearGradient(g) => Brush::LinearGradient(LinearGradientBrush::new(
                g.angle(),
                g.stops()
                    .map(|s| GradientStop { color: s.color.brighter(factor), position: s.position }),
            )),
```